### ISSUE TITLE
ebon gargoyle autoattack bug

### ISSUE THREAD

**Issue by danniellos94** [SunwellTracker/issues#3349]
Decription: 

Ebon gargoyle is doing autoattacks.

How it works: 
https://imgur.com/a/psVsY5H

How it should work: 
He should only cast gargoyle strike.

Source (you should point out proofs of your report, please give us some source):

**Comment by MaruMichal** [comment]
Nope, case closed.

[](https://imgur.com/ctNPuwe)

**Comment by danniellos94** [comment]
It was a popular bug on retail that summon gargoyle were autoattacking a target, it could be avoided by turning your back to boss and sumonning him anyways, it's not a part of his kit his only ability is gargoyle strike.
I'll quote a dev from other server 
"The auto attack is a bug that blizzard had, when the gargoyle was summoned while facing toward the boss it would do one auto attack and interrupt its rotation as well as delay the next cast. This could be remedied by turning away from the boss before summoning the gargoyle. Since we have no idea what kind of spaghetti code caused such behavior we're not going to try to reproduce it either. Especially since it was also avoidable with a trick on retail.

This also makes the number of gargoyle strikes in these specific logs irrelevant since its rotation was interrupted with that auto attack. Gargoyle strikes scale with haste so the number of strikes will greatly vary depending on the players stats when its summoned "

**Comment by danniellos94** [comment]
And here some info about gargoyle with dk having 0 haste
http://www.wowhead.com/forums&topic=133584/best-time-to-summon-gargoyle-procs-unholy-pres
"No raid buffs, only HoW, procs were a bit random, so I only summoned the gargoyle when no procs were up.

Strength 1719, Attack Power 4384. No diseases on target.

In Blood Pres, over its duration it did:
11 Gargoyle Strikes (ave=1788, all hits, no crits), 1 Melee (glancing=409). Total damage done=20077. Damage per second of 692.

In Unholy Pres:
11 Gargoyle Strikes (ave=1748, all hits, no crits), 1 Melee (parried, I was behind dummy, gargoyle appeared in front). Total damage done=19223. Damage per second of 660.

I did a total of 3 tests in unholy, once it managed 12 Gargoyle Strikes, but missed 2 of them.
" 
So with blood pressence and 0 haste and without a bug he should do only 12 casts.
